### DISCUSSION TITLE
In Japan, there are 500 errors in Github.

### DISCUSSION THREAD

**Discussion by y-yamamoto-github** [community/community#116921] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We have confirmed an event in Japan (possibly worldwide) where error number 500 occurs and the page is not displayed.
I am not the only one, but I have also confirmed other articles that are doing this on X (formerly Twitter).
<img width="1440" alt="スクリーンショット 2024-04-05 17 34 31" src="https://github.com/community/community/assets/65648288/596a387c-72d4-4741-b9e3-52c7db17f683">

 Time of event confirmation: 18:00 Japan time (UTC 9:00 am)
Occurrence Condition: After logging in, the corporate repository list is opened.

**Comment by DeathKnight813** [comment]
?

**Comment by cedr0u** [comment]
The error code 500 corresponds to a server that encountered an unexpected issue preventing it from responding to the request. This error response is a generic "catch-all" response.
